src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -182,6 +183,9 @@ public async Task<IList<PublicationTreeThemeViewModel>> GetPublicationTree()
                 : null,
             LatestReleaseId = publication.LatestPublishedReleaseId!.Value,
             IsSuperseded = isSuperseded,
+            SupersededById = publication.SupersededBy?.Id,
+            SupersededByTitle = publication.SupersededBy?.Title,
+            SupersededBySlug = publication.SupersededBy?.Slug,
```

Comment:
I think we need some additional testing in `PublicationServiceTests.cs`, that checks that `SupersededById/Title/Slug` are being returned correctly when calling `PublicationService#Get`.

Reply:
Added `Get_IsSuperseded_SupersededPublicationBuildsPublicationSupersededByViewModel` to `PublicationServiceTests.cs`